_security/configuration/opensearch-keystore.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: default
+title: OpenSearch keystore
+parent: Configuration
+nav_order: 50
+---
+
+# OpenSearch keystore
```

Comment:
Should we change this and the title to "OpenSearch keystore script"?

Reply:
IMO it may make sense to have a dropdown for documentation of all OpenSearch scripts [scripts](https://github.com/opensearch-project/OpenSearch/tree/main/distribution/src/bin) and then include pages for `OpenSearch keystore`, `OpenSearch plugin`, etc. I included this in security configuration because its related to storing secrets securely (i.e. encrypted in the keystore instead of in plain text in files)

Note, this isn't a feature of the security plugin, this is part of the core of OpenSearch.

Reply:
Lets keep this here for now. I think people are more likely to look for this in the security documentation versus a "script" section, but definitely something to keep in mind for later.